teuthology/task/install/util.py

```diff
@@ -109,19 +105,43 @@ def ship_utilities(ctx, config):
                         dst,
                     ],
                 )
+    return filenames
 
-    try:
+def _remove_utilities(ctx, filenames):
```

Comment:
Nit: just to remain consistent, perhaps add docstring

Reply:
Done, thanks.